File: Kotlin/1049.kt
import java.util.Scanner

fun main() {
    val input = Scanner(System.`in`);
    val animal1 = input.nextLine();
    val animal2 = input.nextLine();
    val animal3 = input.nextLine();

    if (animal1 == "vertebrado") {
        if (animal2 == "ave") {
            if (animal3 == "carnivoro") {
                println("aguia")
            } else {
                println("pomba")
            }
        } else {
            if (animal3 == "onivoro") {
                println("homem")
            } else {
                println("vaca")
            }
        }
    } else {
        if (animal2 == "inseto") {
            if (animal3 == "hematofago") {
                println("pulga")
            } else {
                println("lagarta")
            }
        } else {
            if (animal3 == "hematofago") {
                println("sanguessuga")
            } else {
                println("minhoca")
            }
        }
    }
    
}
    

File: Php/1049.php
<?php
$animal1 = readline();
$animal2 = readline();
$animal3 = readline();

if ($animal1 == "vertebrado") {
    if ($animal2 == "ave") {
        if ($animal3 == "carnivoro") {
            print "aguia" . "\n";
        }
        else {
            print "pomba" . "\n";
        }
    }
    else {
        if ($animal3 == "onivoro") {
            print "homem" . "\n";
        }
        else {
            print "vaca" . "\n";
        }
    }
}
else {
    if ($animal2 == "inseto") {
        if ($animal3 == "hematofago") {
            print "pulga" . "\n";
        }
        else {
            print "lagarta" . "\n";
        }
    }
    else {
        if ($animal3 == "hematofago") {
            print "sanguessuga" . "\n";
        }
        else {
            print "minhoca" . "\n";
        }
    }
}


?>